core/arch/riscv64/memfuncs.asm

```diff
@@ -44,16 +44,30 @@ START_FILE
  */
```

Comment:
nit re the title: drop the parenthesis, just "Implemented a naive version of memfuncs" is cleaner.

Reply:
Done.